rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/metadata/TbAbstractGetAttributesNode.java

```diff
@@ -86,97 +91,105 @@ private void safePutAttributes(TbContext ctx, TbMsg msg, T entityId) {
             ctx.tellNext(msg, FAILURE);
             return;
         }
+        JsonNode msgDataNode = JacksonUtil.toJsonNode(msg.getData(), JacksonUtil.ALLOW_UNQUOTED_FIELD_NAMES_MAPPER);
```

Comment:
```
        JsonNode msgDataNode;
        if (fetchToData) {
            msgDataNode = JacksonUtil.toJsonNode(msg.getData(),JacksonUtil.ALLOW_UNQUOTED_FIELD_NAMES_MAPPER)
            if (!msgDataNode.isObject()) {
                ctx.tellFailure(msg, new IllegalArgumentException("Msg body is not an object!"));
                return;
            }
        } else {
            msgDataNode = null;
        }
```

Reply:
Usage of JacksonUtil.ALLOW_UNQUOTED_FIELD_NAMES_MAPPER is redundant in this call, because msg.getData() is always a valid JSON.